recipes/itk-dev/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "4.11.0" %}
+
+package:
+  name: itk-dev
```

Comment:
As this package contains static libraries, and headers it is not needed for run-time usage. It seemed more like a linux `devel` package. If there is not existing convention for this type of package, just `itk` is fine if that is the practice for condo-forge packages.